lib/node_modules/@stdlib/fs/append-file/README.md

```diff
@@ -0,0 +1,255 @@
+
+## Notes
+
+<!-- run-disable -->
+
+```javascript
+var fs = require( 'fs' );
+
+/* eslint-disable node/no-sync */
+
+// Check for directory path existence to prevent an error being thrown...
+if ( fs.existsSync( '/path/to' ) ) {
+    fs.appendFileSync( '/path/to/file.txt', 'beep boop\n' );
+}
+
+/* eslint-enable */
+
+```
+
+can be replaced by an approach which addresses existence via `error` handling.
+
+<!-- run-disable -->
+
+```javascript
+var appendFile = require( '@stdlib/fs/append-file' );
+
+// Explicitly handle the error...
+try {
+    appendFile.sync( '/path/to/file.txt', 'beep boop\n' );
+} catch ( error ) {
+    console.log( error.message );
+}
+```
+
+</section>
+
+<!-- /.notes -->
+
+<section class="examples">
+
+## Examples
+
+<!-- eslint no-undef: "error" -->
+
+```javascript
+var join = require( 'path' ).join;
+var appendFile = require( '@stdlib/fs/append-file' );
+
+var fpath = join( __dirname, 'examples', 'fixtures', 'file.txt' );
+
+// Synchronously append data to a file:
+try {
+    appendFile.sync( fpath, 'beep boop\n', 'utf8' );
+    // throws no errors
+} catch ( error ) {
+    console.log( error instanceof Error );
+    // => false
+}
+
+// Asynchronously append data to a file:
+appendFile( fpath, 'beep boop\n', onAppend );
+
+function onAppend( error ) {
+    if ( error ) {
+        console.error( 'Error: %s', error.message );
+    }
+    console.log( 'Success!' );
+}
+```
+
+</section>
+
+<!-- /.examples -->
+
+* * *
+
+<section class="cli">
+
+## CLI
+
+<section class="usage">
+
+### Usage
+
+```text
+Usage: append-file [options] <filepath>
+
+Options:
+
+  -h,    --help                Print this message.
+  -V,    --version             Print the package version.
+  --enc, --encoding encoding   Encoding. Default: 'utf8'.
+         --flag flag           Flag. Default: 'a'.
+         --mode mode           Mode. Default: 0o666.
+```
+
+</section>
+
+<!-- /.usage -->
+
+<section class="notes">
+
+### Notes
+
+-   Relative output file paths are resolved relative to the current working directory.
+-   Errors are written to `stderr`.
+-   File contents should be provided over `stdin` as part of a [standard stream][standard-stream] pipeline.
+
+</section>
+
+<!-- /.notes -->
+
+<section class="examples">
+
+### Examples
+
+```bash
+$ printf 'beep boop\n' | append-file ./examples/fixtures/file.txt
+```
+
+</section>
+
+<!-- /.examples -->
+
+</section>
+
+<!-- /.cli -->
+
+<!-- Section for related `stdlib` packages. Do not manually edit this section, as it is automatically populated. -->
+
+<section class="related">
+
+* * *
+
+## See Also
+
+-   <span class="package-name">[`@stdlib/fs/exists`][@stdlib/fs/exists]</span><span class="delimiter">: </span><span class="description">test whether a path exists on the filesystem.</span>
+-   <span class="package-name">[`@stdlib/fs/read-file`][@stdlib/fs/read-file]</span><span class="delimiter">: </span><span class="description">read the entire contents of a file.</span>
```

Comment:
```suggestion
```

As stated in the comment, don't manually edit this section.

Reply:
fixed, now this should be pre populated